distributed/utils_test.py

```diff
@@ -1880,16 +1882,16 @@ def check_instances():
 
 
 @contextmanager
-def _reconfigure():
+def default_test_config(**extra_config):
```

Comment:
nit: I'm not a fan of this name since it sounds like this would be the default for basically all tests, but then again I currently lack a better one. If you happen to have an idea for a more descriptive name, I'd be happy, otherwise, leave it as is.

Reply:
@hendrikmakait I renamed to `config_for_cluster_tests` and added a little docstring, what do you think?